### PR TITLE
feat(argo-workflows): Adding initialDelay param to controller configmap

### PR DESCRIPTION
**Pull request by brianderdiarian**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo-workflows/Chart.yaml [hunk 1]
```diff
@@ -16,3 +16,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Add controller namespaceParallelism param to configmap"
+    - "[Added]: Add controller initialDelay param to configmap"
```
Comment (vladlosev) [inline charts/argo-workflows/Chart.yaml:19]: The changes here should describe the current updates not cumulative ones.
So they should replace previous changes, not add to them.

Comment (brianderdiarian) [reply charts/argo-workflows/Chart.yaml:18]: Thanks @vladlosev.  I just removed the redundant changelog entry.

Comment (brianderdiarian) [reply charts/argo-workflows/Chart.yaml:18]: @vladlosev, let me know if you'd like me to bump version again to pass lint check or if you'll do so before merging.  Seems it slipped in the time passed.

Comment (vladlosev) [reply charts/argo-workflows/Chart.yaml:18]: Yes, please do.

Comment (brianderdiarian) [reply charts/argo-workflows/Chart.yaml:18]: Thanks.  Ticked up chart version, `chart-test` workflow can now be rerun.

**Review by vladlosev — APPROVED** [review]
LGTM

**Pull request merged**